frost/bip340.go

```diff
@@ -80,17 +83,13 @@ func (bc *Bip340Curve) IsPointOnCurve(p *Point) bool {
 
 // SerializedPointLength returns the byte length of a serialized curve point.
 func (b *Bip340Curve) SerializedPointLength() int {
-	// From the Marshal() function of secp256k1 go-ethereum implementation:
-	// 	 byteLen := (BitCurve.BitSize + 7) >> 3
-	//   ret := make([]byte, 1+2*byteLen)
-	return 65
+	byteLen := (b.BitSize + 7) >> 3
+	return 1 + 2*byteLen
 }
 
 // SerializePoint serializes the provided elliptic curve point to bytes.
 // The slice length is equal to SerializedPointLength().
 func (b *Bip340Curve) SerializePoint(p *Point) []byte {
```

Comment:
Probably not in-scope of this PR but we are not checking whether the given point is on the curve here. If this is ok,  it's worth flagging this in the method's docstring. This may cause unexpected errors if we ever attempt to marshal a point whose coordinates use a different byte size than the BIP340 curve so, worth flagging.